src/main/resources/data/computercraft/lua/rom/apis/textutils.lua

```diff
@@ -645,9 +652,17 @@ end
 -- @throws If the object contains a value which cannot be
 -- serialised. This includes functions and tables which appear multiple
 -- times.
-function serialize(t)
+function serialize(t, opts)
     local tTracking = {}
-    return serializeImpl(t, tTracking, "")
+		expect(2, opts, "table", "nil")
+		if opts then
+			field(opts, "noWrap", "boolean", "nil")
```

Comment:
Possibly worth switching to snake_case - that's what we do with `unserializeJSON`.

I'd also make all the terms positive, so you'd have `{ wrap = false }` instead of `{ noWrap = true }`. It gets a little confusing if you're specifying options explicitly - the double negative of `{ noWrap = false }` is a little hard to parse :).